File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
     
        vector<bool> arr2;
        
        
int max=*max_element(candies.begin(), candies.end());        
        
        for(int i=0;i<candies.size();i++)
        {
                if(candies[i]+extraCandies>=max)
                    arr2.push_back(true);
                else 
                    arr2.push_back(false);
            }
        
        return arr2;
        
    }
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        
        max1=max(candies)
        lis1=[]
        
        for i in candies:
            
            if i+extraCandies>=max1:
                lis1.append(True)
            else:
                lis1.append(False)
        return lis1